mmv1/third_party/terraform/services/bigtable/resource_bigtable_table.go

```diff
@@ -335,6 +401,36 @@ func resourceBigtableTableUpdate(d *schema.ResourceData, meta interface{}) error
 		}
 	}
 
+	if d.HasChange("automated_backup_policy") {
+		abpField := d.Get("automated_backup_policy")
+		if abpField == nil {
```

Comment:
I probably got confused about what we decided, I thought we want to follow CDC and disable when retention_period and frequency is set as 0, and no change if automated_backup_policy is null.

Reply:
Discussed offline, this is implementing the right behavior defined in https://docs.google.com/document/d/1NzFf9Fi3MLJFG4sZgeVhbxMlxbKumF7xPQ1U3mgquJs, i.e. nill to disable.

Reply:
Thanks. On further thought, I believe we should continue with 0-to-disable 

I've updated the code here

I've also posted a PR here: https://github.com/googleapis/google-cloud-go/pull/10226